lightning/src/chain/keysinterface.rs

```diff
@@ -80,7 +80,7 @@ pub enum SpendableOutputDescriptor {
 	/// To derive the remote_revocation_pubkey provided here (which is used in the witness
```

Comment:
This still says remote_, as does the variable name further down.

Reply:
Updated all ChannelMonitor/OnchainTxHandler-side, as it was tied down with using this version of Remote.